ghost/email-service/lib/EmailRenderer.js

```diff
@@ -215,11 +215,7 @@ class EmailRenderer {
         if (newsletter.get('sender_reply_to') === 'support') {
             return this.#settingsHelpers.getMembersSupportAddress();
         }
-        if (newsletter.get('sender_reply_to') === 'newsletter') {
-            if (this.#emailAddressService.managedEmailEnabled) {
-                // Don't duplicate the same replyTo addres if it already in the FROM address
-                return null;
-            }
+        if (newsletter.get('sender_reply_to') === 'newsletter' && !this.#emailAddressService.managedEmailEnabled) {
```

Comment:
This works, but the `this.#emailAddressService.getAddress` will log an error if the replyTo address is invalid. So if it is set to `newsletter` and we continue after this if check to the `this.#emailAddressService.getAddress` part, it will log an error every time. Ideally we should set replyTo to undefined when that is the case and avoid that error log

Reply:
Not blocking a merge though

Reply:
Thanks! Addressed in a498ceba15274a08be5fc57b798d722510398e97